clients/docs/ccloud.rst

```diff
@@ -0,0 +1,229 @@
+.. _client-examples-ccloud:
+
+Confluent Cloud CLI
+-------------------
+
+In this tutorial, you will run a |confluent-cli| client application that
+produces messages to and consumes messages from an |ak-tm| cluster.
+
+.. include:: includes/client-example-overview.rst
+
+
+Prerequisites
+-------------
+
+Client
+~~~~~~
+
+-  Local install of :ref:`Confluent Cloud CLI <ccloud-install-cli>` v1.13.0 or later.
+
+Kafka Cluster
+~~~~~~~~~~~~~
+
+.. include:: includes/client-example-prerequisites.rst
+
+
+Setup
+-----
+
+#. .. include:: includes/clients-checkout.rst
+
+#. Change directory to the example for |confluent-cli|.
+
+   .. code-block:: bash
+
+      cd clients/cloud/ccloud/
+
+#. .. include:: includes/client-example-create-file.rst
+
+
+Basic Producer and Consumer
+----------------------------
+
+.. include:: includes/producer-consumer-description.rst
+
+
+Produce Records
+~~~~~~~~~~~~~~~
+
+.. Should steps 1 through 3 below be under this  "Produce Records" section or before it?
+
+#. Create the topic in |ccloud|.
+
+   .. code-block:: bash
+
+      ccloud kafka topic create test1
+
+#. Run the :ref:`Confluent Cloud CLI producer <ccloud_kafka_topic_produce>`,
+   writing messages to topic ``test1``, passing in additional arguments:
+
+   -  ``--parse-key --delimiter ,``: pass key and value, separated by a comma
+
+   .. code-block:: bash
+
+      ccloud kafka topic produce test1 --parse-key --delimiter ,
+
+.. Is there any output the user should verfiy when running the previous command or expected output they should see?
+
```

Comment:
@ybyzek Is there any output the user should verify when running the previous command or expected output they should see?

Reply:
Not for produce

Reply:
Got it